Zend/zend_API.h

```diff
@@ -1482,6 +1487,9 @@ ZEND_API ZEND_COLD void ZEND_FASTCALL zend_argument_value_error(uint32_t arg_num
 #define Z_PARAM_OBJECT_OF_CLASS(dest, _ce) \
 	Z_PARAM_OBJECT_OF_CLASS_EX(dest, _ce, 0, 0)
 
+#define Z_PARAM_OBJECT_OF_CLASS_OR_NULL(dest, _ce) \
+	Z_PARAM_OBJECT_OF_CLASS_EX(dest, _ce, 1, 0)
```

Comment:
Please commit these separately, together with a conversion of any existing usages of _EX in the codebase, if any.

Reply:
Sure!

Reply:
There were not so many instances. Although I left out a couple of FFI methods that make use of the separation parameter of `Z_PARAM_OBJECT_OF_CLASS_EX2`.  `Z_PARAM_OBJECT_OF_CLASS_SEPARATED` sound a bit weird and not a very widespread usecase.

Reply:
Yeah, I don't think we need separate macros for the FFI case, which does something it really, really shouldn't be doing...